### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by vladislavantonov-react**
Modal

### CODE REVIEW

File: src/components/Modal/modal.module.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  width:35%;
+}
+.control{
+  width: 50% !important;
+  display: inline-flex !important;
+}
+.price {
+  background-color: #eb7413 !important;
+  border-color: #eb7413 !important;
+  .plus{
```
Comment (justiceteam-it) [inline src/components/Modal/modal.module.scss:11]: remove important

File: src/components/Modal/Modal.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styles from './modal.module.scss';
+import {Modal, Button,Form} from 'react-bootstrap';
+import {FaCartPlus} from 'react-icons/fa';
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
```
Comment (justiceteam-it) [inline src/components/Modal/Modal.jsx:6]: fix it

File: src/index.js [hunk 3]
```diff
@@ -5,9 +5,9 @@ import App from './App';
 import 'bootstrap/dist/css/bootstrap.min.css';
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  // <React.StrictMode>
+    <App />,
+  // </React.StrictMode>,
```
Comment (justiceteam-it) [inline src/index.js:10]: remove comments

File: src/layouts/Home/Home.jsx [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
-import lampTwo from '../../assets/image/lampTwo.png'
-import lampThree from '../../assets/image/lampThree.png'
-import lampFour from '../../assets/image/lampFour.png'
+import lampOne from '../../assets/image/lampOne.png';
+import lampTwo from '../../assets/image/lampTwo.png';
+import lampThree from '../../assets/image/lampThree.png';
+import lampFour from '../../assets/image/lampFour.png';
 import Container from "react-bootstrap/Container";
 import Col from "react-bootstrap/Col";
 import Row from "react-bootstrap/Row";
```
Comment (justiceteam-it) [inline src/layouts/Home/Home.jsx:10]: fix it

File: src/layouts/Home/Home.jsx [hunk 5]
```diff
@@ -42,7 +46,7 @@ const Home = () => {
                     <Row className="justify-content-space-between d-flex">
                     {arr.map((item, key) => (
                         <Col lg={6} key={key}>
-                            <ImgMediaCard title={item.title} price={item.price} img={item.img}/>
+                            <ImgMediaCard title={item.title} price={item.price} img={item.img} about={item.about}/>
```
Comment (justiceteam-it) [inline src/layouts/Home/Home.jsx:49]: use column

File: src/layouts/SignUp/SignUp.jsx [hunk 6]
```diff
@@ -1,5 +1,5 @@
 import React, {useEffect, useState} from 'react';
-import styles from './signup.module.scss'
+import styles from './signup.module.scss';
 import {Container, Form, Button} from "react-bootstrap";
```
Comment (justiceteam-it) [inline src/layouts/SignUp/SignUp.jsx:3]: same

File: .idea/workspace.xml [hunk 7]
```diff
@@ -5,18 +5,15 @@
   </component>
   <component name="ChangeListManager">
     <list default="true" id="c4830c36-7749-4124-844f-577c475201a7" name="Default Changelist" comment="">
-      <change afterPath="$PROJECT_DIR$/src/components/Header/header.module.scss" afterDir="false" />
+      <change afterPath="$PROJECT_DIR$/src/components/Modal/Modal.jsx" afterDir="false" />
+      <change afterPath="$PROJECT_DIR$/src/components/Modal/modal.module.scss" afterDir="false" />
       <change beforePath="$PROJECT_DIR$/.idea/workspace.xml" beforeDir="false" afterPath="$PROJECT_DIR$/.idea/workspace.xml" afterDir="false" />
-      <change beforePath="$PROJECT_DIR$/src/App.jsx" beforeDir="false" afterPath="$PROJECT_DIR$/src/App.jsx" afterDir="false" />
       <change beforePath="$PROJECT_DIR$/src/components/Card/Card.jsx" beforeDir="false" afterPath="$PROJECT_DIR$/src/components/Card/Card.jsx" afterDir="false" />
```
Comment (justiceteam-it) [inline .idea/workspace.xml]: remove idea dir

File: src/components/Card/Card.jsx [hunk 8]
```diff
@@ -1,28 +1,39 @@
 import React from "react";
-import styles from "./card.module.scss"
-
+import styles from "./card.module.scss";
```
Comment (justiceteam-it) [inline src/components/Card/Card.jsx:2]: fix it

**Review by justiceteam-it — CHANGES_REQUESTED** [review]

File: src/components/Modal/modal.module.scss (hunk 1 above)
Comment (vladislavantonov-react) [reply src/components/Modal/modal.module.scss:11]: i will remove important  when i implement cart

**Pull request merged**